hub/core/tensor.py

```diff
@@ -1058,3 +1143,9 @@ def list(self):
             return list(self.numpy())
         else:
             return list(map(list, self.numpy(aslist=True)))
+
+    @staticmethod
+    def _check_whether_sample_info_is_empty(sample_info):
```

Comment:
this can be a utility function and not a static method.

Reply:
most probably we don't need this at all, it can be just one line, inside the body instead of a function/method
return len(sample_info) == 0